www/js/app.js

```diff
@@ -464,6 +464,16 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         settingsStoreStatusDiv.classList.remove('apiAvailable', 'apiUnavailable');
         settingsStoreStatusDiv.classList.add(params.storeType === 'none' ? 'apiUnavailable' : 'apiAvailable');
         apiPanelClass = params.storeType === 'none' ? 'card-warning' : apiPanelClass;
+        // Update Decompressor API section of panel
+        var decompAPIStatusDiv = document.getElementById('decompressorAPIStatus');
+        apiName = params.decompressorAPI.assemblerMachineType;
+        if (apiName && params.decompressorAPI.decompressorLastUsed) {
+            apiName += ' [&nbsp;' + params.decompressorAPI.decompressorLastUsed + '&nbsp;]';
+        }
+        apiPanelClass = params.decompressorAPI.errorStatus ? 'card-danger' : apiName ? apiPanelClass : 'card-warning';
```

Comment:
I'm wary of cluttering the API panel, as I think each API's info should fit on a single line at least in an average desktop browser. Therefore I'd be more in favour of using the orange colour for ASM. The panel is only really for people who understand something about HTML/JS, and those people will know what these messages mean (or can find out easily if they are interested). The average user will never look at this panel.

Reply:
OK I agree

Reply:
I've implemented turning the colour orange (both the text and the panel) if the API is 'ASM', for which I've added a new class name `apiSuboptimal` in `app.css`. I've done a quick test with IE11, artificially setting the Service Worker availability to true in order to test that this class setting works, and it seems to. Ideally we'd need to test on a broswer that has Service Worker, but does not have WASM... Not sure if there is one.

Reply:
I tested on my Firefox OS device : it displays as expected "Decompressor API: ASM [ ... ]" in orange, as it does not support WASM.
BUT it's also in orange on Chromium and Firefox, with "Decompressor API: WASM [ ... ]"
It's probably because you test that `apiName === 'WASM'`, where there can be a suffix [ ... ] : I suppose it should be a "starts with" test instead of equality

Reply:
Apologies, I did this in a rush this afternoon and didn't actually test loading a ZIM... Will correct early tomorrow, so we can merge before release.

Reply:
Perfect, no problem!

Reply:
Hopefully now fixed. I simply moved the place where the decompressor type is added to the API name, so `apiName = params.decompressorAPI.assemblerMachineType` at the point of testing.